L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker.cc

```diff
@@ -1272,12 +1272,12 @@ void L1TrackObjectNtupleMaker::analyze(const edm::Event& iEvent, const edm::Even
   iEvent.getByToken(GenParticleToken_, GenParticleHandle);
 
   //Vertex
-  edm::Handle<l1t::VertexCollection> L1TkPrimaryVertexHandle;
-  iEvent.getByToken(L1VertexToken_, L1TkPrimaryVertexHandle);
+  edm::Handle<l1t::VertexCollection> L1PrimaryVertexHandle;
+  iEvent.getByToken(L1VertexToken_, L1PrimaryVertexHandle);
   std::vector<l1t::Vertex>::const_iterator vtxIter;
 
-  edm::Handle<l1t::VertexWordCollection> L1TkPrimaryVertexEmuHandle;
-  iEvent.getByToken(L1VertexEmuToken_, L1TkPrimaryVertexEmuHandle);
+  edm::Handle<l1t::VertexWordCollection> L1PrimaryVertexEmuHandle;
+  iEvent.getByToken(L1VertexEmuToken_, L1PrimaryVertexEmuHandle);
```

Comment:
since you are already touching these, could you change the local variable names to lowercase ?

Reply:
I am happy for these to go to lowercase to follow CMSSW style rules